core/templates/pages/topic-editor-page/services/create-new-skill-modal.service.spec.ts

```diff
@@ -122,19 +122,24 @@ describe('Create New Skill Modal Service', () => {
   beforeEach(() => {
     createNewSkillModalService = TestBed.inject(CreateNewSkillModalService);
     alertsService = TestBed.inject(AlertsService);
-    alertsService = (alertsService as unknown) as
+    alertsService = (alertsService) as
```

Comment:
```suggestion
    alertsService = alertsService as jasmine.SpyObj<AlertsService>;
```

Reply:
ditto below, get rid of the unnecessary branckets

Reply:
Done